script/Deploy.s.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.21;
+
+import "forge-std/Script.sol";
+import "../src/WorldIDRouter.sol";
+import "../src/WorldIDRouterImplV1.sol";
+import "../src/WorldIDIdentityManager.sol";
+import "../src/WorldIDIdentityManagerImplV1.sol";
+import "../src/WorldIDIdentityManagerImplV2.sol";
+import "../src/SemaphoreVerifier.sol";
+
+import {Verifier as InsertionB10} from "../src/verifiers/insertion/b10.sol";
+import {Verifier as InsertionB100} from "../src/verifiers/insertion/b100.sol";
+import {Verifier as InsertionB600} from "../src/verifiers/insertion/b600.sol";
+import {Verifier as InsertionB1200} from "../src/verifiers/insertion/b1200.sol";
+
+import {Verifier as DeletionB10} from "../src/verifiers/deletion/b10.sol";
+import {Verifier as DeletionB100} from "../src/verifiers/deletion/b100.sol";
+
+contract Deploy is Script {
+    uint8 constant TREE_DEPTH = 30;
+    uint256 constant INITIAL_ROOT =
+        0x918D46BF52D98B034413F4A1A1C41594E7A7A3F6AE08CB43D1A2A230E1959EF;
+
+    address semaphoreVerifier = address(0);
+
+    address batchInsertionVerifiers = address(0);
+    address batchDeletionVerifiers = address(0);
+
+    function run() external returns (address router, address worldIDOrb, address worldIDPhone) {
+        console.log("Deploying WorldIDRouter, WorldIDOrb, and WorldIDPhone");
+
+        WorldIDIdentityManager worldIDOrb = deployWorldID(INITIAL_ROOT);
+        console.log("WorldIDOrb:", address(worldIDOrb));
+        WorldIDIdentityManager worldIDPhone = deployWorldID(INITIAL_ROOT);
```

Comment:
Do we need to deploy the phone credential if the phone sequencer is offchain?

Reply:
Good point, we don't!